src/reverse-numbers.js

```diff
@@ -0,0 +1,11 @@
+function reverseNumbers(numbers) {
```

Comment:
delele this due to miss understand requirement

Reply:
okay, I did.